bp/src/BPConfiguration.m

```diff
@@ -110,6 +110,8 @@ typedef NS_OPTIONS(NSUInteger, BPOptionType) {
         "Enable verbose logging"},
     {'k', "keep-individual-test-reports", BP_MASTER | BP_SLAVE, NO, NO, no_argument, "Off", BP_VALUE | BP_BOOL, "keepIndividualTestReports",
         "Keep individual test reports, in addition to the aggregated final report"},
+    {'Z', "unsafe-skip-xcode-check", BP_MASTER | BP_SLAVE, NO, NO, no_argument, "Off", BP_VALUE | BP_BOOL , "unsafeSkipXcodeCheck",
```

Comment:
1. `unsafe-skip-xcode-version-check` might be better unless you insist on keeping it short.
2. You might want to add this option in `    // options without short-options` section instead of choosing `Z` which doesn't sound very relevant, as a short-option.
3. Also, can you please add this option to `README.md`.